runway/data_types.py

```diff
@@ -646,3 +644,88 @@ def to_dict(self):
         ret = super(boolean, self).to_dict()
         ret['default'] = self.default
         return ret
+
+
+class point(BaseType):
+    """A point data type representing a specific location in an image. 
+    It accepts two normalized floating point numbers [x, y] between 0 and 1,
+    where [0, 0] represents the top-left corner and [1, 1] the bottom-right corner of an image.
+    
+    .. code-block:: python
+
+        import runway
+        from runway.data_types import image, point
+
+        @runway.command('detect_gaze', inputs={'image': image()}, outputs={'gaze_location': point()})
+        def detect_gaze(model, inputs):
+            result = model.run(inputs['image'])
+            return {'gaze_location': result}
+
+    :param description: A description of this variable and how it's used in the model,
+        defaults to None
+    :type description: string, optional
+    """
+    def __init__(self, description=None):
+        super(point, self).__init__('point', description=description)
+    
+    def validate(self, value):
```

Comment:
I added a similar validate function when I created the `boolean` data type. I like this general purpose `void` validate function that throws `InvalidArgumentErrors` if there are any validation errors. Do you think it might make sense to add such a function to BaseType (not in this PR of course). It could even be a decorator that can be used to wrap `serialize()` and `deserialize()`.

Reply:
👍 Yes, I'd be in favor of that.